packages/shared/lib/src/sdk/tx.rs

```diff
@@ -70,7 +71,11 @@ impl SigningData {
             None => None,
         };
 
-        let fee_payer = signing_tx_data.fee_payer.to_string();
+        let fee_payer = match signing_tx_data.fee_payer {
+            Either::Left(f) => f.0.to_string(),
+            Either::Right(_) => panic!("Fee payer must be present in SigningData"),
```

Comment:
You can use this:
```rs
            Either::Right(_) => return Err(JsError::new("Fee payer must be a public key")),
```
panics in wasm can leak memory :rip:

Reply:
thanks!